lib/gems/pending/appliance_console/locales/appliance/en.yml

```diff
@@ -25,10 +22,7 @@ en:
     - shutdown
     - summary
     - quit
-    dhcp: Set DHCP Network Configuration
-    static: Set Static Network Configuration
-    testnet: Test Network Configuration
-    hostname: Set Hostname
+    networking: Set-up Networking
```

Comment:
`Set-up` is not proper English.  This should be either `Setup` or `Set up` or just `Set`